### PR TITLE
Upgrade @kyma-project/asyncapi-react to 0.14.2

### PR DESCRIPTION
**Pull request by Fox32**
There is an interesting fix for displaying async messages in the docs.

See https://github.com/asyncapi/asyncapi-react/pull/156

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/spotify/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

### CODE REVIEW

File: yarn.lock
Comment (dhenneke) [inline yarn.lock:10351]: I'm not an expert here. But why didn't `yarn` collapsed these three dependencies into a single block that resolves to 2.2.0?

Comment (freben) [reply yarn.lock:10351]: Because that might change the behavior of existing code, I think, so it takes the safe route until you choose to upgrade it all. It's sometimes frustrating. I have sometimes run https://github.com/atlassian/yarn-deduplicate, but that can also be risky.

Comment (dhenneke) [reply yarn.lock:10351]: I didn't mean to block this. I was just curious on why this would happen.

Comment (freben) [reply yarn.lock:10351]: Ah no I didn't take it as such, I was just typing that while not at the computer so I wanted to review today instead :)

**Review by freben — APPROVED** [review]

**Comment by freben** [comment]
@Fox32 one conflict to solve - maybe there will be more after we merge #2654 - please hold off on merging this one as well until after that :) It's due to be merged tomorrow morning

**Review by Rugvip — APPROVED** [review]
🎉

**Comment by Fox32** [comment]
@freben If I resolve the conflict, should I also add a changeset?

**Comment by Rugvip** [comment]
@Fox32 Can be good to add a patch changeset to make sure that it gets released

**Comment by Fox32** [comment]
Done!

**Pull request merged**
